### DISCUSSION TITLE
Suggest repository names while trying to filter by repo

### DISCUSSION THREAD

**Discussion by Jackenmen** [community/community#9780] [Other Feature Feedback, Questions, & Ideas]
Before `cs.github.com`, I was a big user of `grep.app`. After typing out the query, `grep.app`, same as `cs.github.com`, allows you to filter by the repository. But unlike `cs.github.com`, it is much easier to work with as you don't need to know the exact name of the owner and repository - as you type, grep.app further filters the results and suggest the repository names to filter by:
![image](https://user-images.githubusercontent.com/6032823/148546547-349b3097-3ea9-403a-8349-fbfbe415d1e5.png)

Being able to see the results filtered by partial repository name is certainly nice but I think it would be enough if GitHub just showed repository name suggestions as you type so that you don't have to know how exactly the repository is called. And it also could save you some keystrokes :)

**Comment by look** [comment]
Thanks for the feedback. We do suggest repos when you type `repo:` but not until you type `/` which isn't as easy to use. This is definitely on my list of things to improve!
